person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @age = age
+    @name = name
```

Comment:
- [ ] Hi as part of the requirements, you are supposed to initialize the instance variable id too. so ill recommend initializing it with a unique random value
i.e `@id = Random.rand(1..1000)`